airlock/business_logic.py

```diff
@@ -84,6 +84,25 @@ class AuditEventType(Enum):
     REQUEST_FILE_REJECT = "REQUEST_FILE_REJECT"
 
 
+AUDIT_MSG_FORMATS = {
+    AuditEventType.WORKSPACE_FILE_VIEW: "Viewed file",
+    AuditEventType.REQUEST_FILE_VIEW: "Viewed file",
+    AuditEventType.REQUEST_FILE_DOWNLOAD: "Downloaded file",
+    AuditEventType.REQUEST_CREATE: "Created request",
+    AuditEventType.REQUEST_SUBMIT: "Submitted request",
+    AuditEventType.REQUEST_WITHDRAW: "Withdrew request",
+    AuditEventType.REQUEST_APPROVE: "Approved request",
+    AuditEventType.REQUEST_REJECT: "Rejected request",
+    AuditEventType.REQUEST_RELEASE: "Released request",
+    AuditEventType.REQUEST_EDIT: "Edited the Context/Controls",
```

Comment:
```suggestion
    AuditEventType.REQUEST_EDIT: "Edited the group Context/Controls",
```

Reply:
That is the longest string. While your version is clearer, it adds 6 characters to the table column width, and we're tight on horizontal space. We have group in the Context column anyway, I think that's maybe enough?

Reply:
fair point - wdyt about going the other way & dropping `verb group` from the others?

Reply:
done